tests/lib/release-ops.js

```diff
@@ -114,30 +114,68 @@ describe("ReleaseOps", function() {
                 type: "major",
                 changelog: {
                     fix: [
-                        "Fix: Something",
-                        "Fix: Something else"
+                        "* abcd123 Fix: Something (githubhandle)",
+                        "* 321dcba Fix: Something else (Abc D. Efg)"
                     ],
                     docs: [
-                        "Docs: Something else"
+                        "* a1b2c3d Docs: Something else (Committer Name)"
                     ],
                     update: [
-                        "Update: Foo"
+                        "* 9876543 Update: Foo (Tina Tester)"
                     ],
                     breaking: [
-                        "Breaking: Whatever"
+                        "* 1234567 Breaking: Whatever (Toby Testing)"
                     ]
                 },
                 rawChangelog: logs.join("\n")
             });
         });
 
+        it("should disregard reverted commits", function() {
+            var logs = [
+                    "* abcd123 Docs: Update something in the docs (githubhandle)",
+                    "This is the body.",
+                    "It has multiple lines.",
+                    "* a1b2c3d Revert \"Breaking: A breaking change (fixes #1234)\" (Committer Name)",
+                    "This reverts commit abcdef010c481d5da8d2d9b5ef74945e6566166c.",
+                    "This explains why.",
+                    "* 321dcba Fix: Fix a bug (fixes #4321) (Abc D. Efg)",
+                    "Describe the bug.",
+                    "* 9876543 Revert \"New: Add cool new feature (fixes #42)\" (Tina Tester)",
+                    "This reverts commit 123456710c481d5da8d2d9b5ef74945e6566166c.",
+                    "* abcdef0 Breaking: A breaking change (fixes #1234) (Cool Committer)",
+                    "* 1234abc Revert \"New: From a previous release (fixes #1234)\" (Foo Bar)",
+                    "This reverts commit 0123456789abcdeffedcba9876543210a1b2c3d4.",
```

Comment:
What about including the ones that span releases, since those actually contribute to the diff, and still excluding commit/revert pairs that are fully contained within a single release?